src/client/datascience/notebook/integration.ts

```diff
@@ -97,6 +100,15 @@ export class NotebookIntegration implements IExtensionSingleActivationService {
             }
         }
     }
+
+    private registerCompletionItemProvider() {
+        const disposable = languages.registerCompletionItemProvider(
+            { language: PYTHON_LANGUAGE, scheme: NotebookCellScheme },
```

Comment:
In the Python extension we have two different document selectors for Python notebooks:
https://github.com/microsoft/vscode-jupyter/blob/e8af5b35ebba334e5e1a2e521895d3ffee0e4131/src/client/common/constants.ts#L11-L12

Do we need to use both here, or do they serve different purposes?

Reply:
Removed. this was added in the early days when VSC was still working on the schemes for the cells.